easybuild/easyblocks/p/pybind11.py

```diff
@@ -91,13 +91,18 @@ def sanity_check_step(self):
         # don't add user site directory to sys.path (equivalent to python -s)
         env.setvar('PYTHONNOUSERSITE', '1', verbose=False)
         # Get python includes
-        fake_mod_data = self.load_fake_module(purge=True)
+        if not self.is_extension:
+            # for stand-alone Python package installations (not part of a bundle of extensions),
```

Comment:
comment doesn't really match what we are doing here

Reply:
What would you suggest ? I simply copied the same comment as from PythonPackage

Reply:
borrowing what i think is a much more direct and descriptive comment from extensioneasyblock.py in framework:

```python
# only load fake module for stand-alone installations (not for extensions),
# since for extension the necessary modules should already be loaded at this point
if not self.is_extension:
    ...
```

I really don't understand what the the python easyblock means by "missing dependencies"